### PR TITLE
BZ1785383 --added new review feedback

### PR DESCRIPTION
**Pull request by jhoyt-rh**
This is for BZ1785383.

Added reference to  image import requirements, and made edits based on more feedback.

### CODE REVIEW

File: doc-Installing_on_AWS/topics/installation.adoc [hunk 1]
```diff
@@ -169,30 +195,37 @@ To import the appliance:
...
 ----
-$ aws ec2 import-snapshot --disk-containers file://containers.json
+$ aws ec2 import-snapshot --disk-container file://containers.json
 ----
 +
 . Check the progress of your snapshot import by running the following command:
 +
 ----
- $ aws ec2 describe-import-snapshot-tasks --import-task-ids import-ami-AMI_ID_GOT_FROM_RESPONSE
+ $ aws ec2 describe-import-snapshot-tasks --import-task-ids snap-SNAPSHOT_ID_GOT_FROM_RESPONSE
```
Comment (mmojzis) [inline doc-Installing_on_AWS/topics/installation.adoc]: import-snap-SNAPSHOT_ID_GOT_FROM_RESPONSE or just SNAPSHOT_ID_GOT_FROM_RESPONSE

**Comment by simaishi** [comment]
I think what we provide (and upload to S3) should be called "image" or "disk". That will be imported to AWS as "snapshot".

File: doc-Installing_on_AWS/topics/installation.adoc (hunk 1 above)
Comment (jhoyt-rh) [reply doc-Installing_on_AWS/topics/installation.adoc]: Changed to second option: "SNAPSHOT_ID_GOT_FROM_RESPONSE"

**Comment by jhoyt-rh** [comment]
Verified the manual contains the updates. Merging this PR.

**Pull request merged**